Packs/Core/Playbooks/playbook-A_source_host_created_uncommon_remote_scheduled_task.yml

```
@@ -0,0 +1,1344 @@
+        simple: 3 hours ago
+      query:
+        simple: (mitreattcktechnique:*T1202* or mitreattcktechnique:*T1021*) and -name:"Uncommon remote scheduled task created" and agentid:${Core.Endpoint.endpoint_id}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1150,
+          "y": 1805
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "74":
+    id: "74"
+    taskid: 4be27c4b-ad83-46b4-868b-795a35647cd0
+    type: regular
+    task:
+      id: 4be27c4b-ad83-46b4-868b-795a35647cd0
+      version: -1
+      name: Get remote endpoint details
+      description: Gets a list of endpoints, according to the passed filters. If there are no filters, all endpoints are returned. Filtering by multiple fields will be concatenated using the AND condition (OR is not supported). Maximum result set size is 100. Offset is the zero-based number of the endpoint from the start of the result set (start by counting from 0).
+      script: '|||core-get-endpoints'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "29"
+    scriptarguments:
+      hostname:
+        simple: ${Core.OriginalAlert.raw_abioc.event.schtasks_remote_host}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 410,
+          "y": 70
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "75":
+    id: "75"
+    taskid: a2bad801-d912-45ec-8ac6-5ece49400caa
+    type: condition
+    task:
+      id: a2bad801-d912-45ec-8ac6-5ece49400caa
+      version: -1
+      name: Is the XSIAM agent installed on the remote endpoint?
+      description: Checks whether the Cortex XSIAM agent is installed on the remote endpoint.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "5"
+      "yes":
+      - "73"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Core.Endpoint.endpoint_id
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1150,
+          "y": 1610
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "76":
+    id: "76"
+    taskid: 36164015-8f07-4e5b-873d-024f66adb228
+    type: condition
+    task:
+      id: 36164015-8f07-4e5b-873d-024f66adb228
+      version: -1
+      name: Is the XSIAM agent install and connected on the remote endpoint?
```

Is agent install and connected on the remote endpoint? or XDR agent. 
its not XSIAM agent :)

---

true for more tasks

---

prefered first option, let's try to not mention XDR
```suggestion
      name: is agent install and connected on the remote endpoint?
```

---

Done.